plugins/GSdx/GSRendererHW.cpp

```diff
@@ -207,15 +207,16 @@ int GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
 	return (int)t & ~0xF; // cheap rounding
 }
 
+//X1 not even used?? Should this be similar to alpha1 with different rounding?
 float GSRendererHW::alpha0(int L, int X0, int X1)
 {
-	float x = (X0 + 15) & ~0xF; // Round up
-	return (x - X0) / (float)L;
+	float x = float((X0 + 15) & ~0xF); // Round up
```

Comment:
no alpha0 is the interpolation factor of first pixels. Alpha1 is the interpolation factor os the last pixel. (Note L = X1 - X0). However I think it would be better to cast it this way.
```
	int x = (X0 + 15) & ~0xF; // Round up
	return float(x - X0) / (float)L;
```